src/V3Sched.cpp

```diff
@@ -1223,12 +1234,26 @@ void schedule(AstNetlist* netlistp) {
               ? createTriggerSenTree(netlistp, actTrig.m_vscp, dpiExportTriggerIndex)
               : nullptr;
 
+    std::map<const AstIface*, AstSenTree*> vifTriggeredAct;
+    {
+        size_t vifTriggerIndex = firstVifTriggerIndex;
+        for (const auto& p : netlistp->virtIfaceTriggerps()) {
+            vifTriggeredAct.insert(std::make_pair(
+                p.first, createTriggerSenTree(netlistp, actTrig.m_vscp, vifTriggerIndex)));
+            vifTriggerIndex++;
```

Comment:
```suggestion
            ++vifTriggerIndex;
```

Likewise elsewhere.  BTW please click "resolved" in github when these suggestions are implemented thanks.

Reply:
Can you provide the rationale behind this?

Reply:
Preincrement is faster for some types. So preincrement is always right.  Postincrement is right only if you look into the data types behind it which takes effort/analysis.  So might as well always preincrement.